tests/kubeless-test-client/k8s.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: gateway.kyma.cx/v1alpha2
+kind: Api
+metadata:
+  namespace: kubeless-test
+  name: test-hello
+  labels:
+    function: test-hello
+spec:
+  authentication: []
+  hostname: test-hello.yfactory.sap.corp
```

Comment:
Probably **kyma.local** domain should be used instead of **yfactory.sap.corp**

Reply:
Fixed.